src/amber/cli/templates/app.cr

```diff
@@ -24,5 +32,22 @@ module Amber::CLI
     private def generate_database_name_base
       @name.gsub('-', '_')
     end
+
+    def fetch_author
+      return "[your-name-here]" unless system(WHICH_GIT_COMMAND)
+      `git config --get user.name`.strip
+    end
+
+    def fetch_email
+      return "[your-email-here]" unless system(WHICH_GIT_COMMAND)
+      `git config --get user.email`.strip
+    end
+
+    def fetch_github_name
+      default = "[your-github-name]"
+      return default unless system(WHICH_GIT_COMMAND)
```

Comment:
`system(WHICH_GIT_COMMAND)` will be called 3 times (`getch_author`, `fetch_email`, `fetch_github_name`). Better to check whether git exists or not earlier.

Reply:
Maybe we should change it on crystal itself too?
https://github.com/crystal-lang/crystal/blob/master/src/compiler/crystal/tools/init.cr

Reply:
Sorry, why to change it on crystal report if you propose the change here?

Reply:
@veelenga Done! Thank you for your suggestion! 👍